drivers/sensor/bmp085/Kconfig

```
@@ -0,0 +1,48 @@
+# Bosch BMP085 pressure sensor configuration options
+
+# Copyright (c) 2024, Mario Paja
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig BMP085
+	bool "Bosch BMP085"
```

We tend to typically elaborate on what actual type of sensor this is


```suggestion
	bool "Bosch BMP085 digital pressure sensor"
```